src/code/z_actor.c

```diff
@@ -845,7 +845,24 @@ void Actor_FreeOverlay(ActorOverlay* entry) {
 
 #pragma GLOBAL_ASM("asm/non_matchings/code/z_actor/func_800BE03C.s")
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_actor/func_800BE0B8.s")
+Actor *func_800BE0B8(GlobalContext *globalCtx, Actor *inActor, s16 arg2, u8 arg3, f32 arg4) {
+    Actor *actor = globalCtx->actorCtx.actorList[arg3].first;
```

Comment:
```suggestion
Actor* func_800BE0B8(GlobalContext* globalCtx, Actor* inActor, s16 arg2, u8 arg3, f32 arg4) {
    Actor* actor = globalCtx->actorCtx.actorList[arg3].first;
```
These should have been caught by the formater?